golangci.yml

```diff
@@ -18,10 +18,9 @@ linters:
     - deadcode
     - errcheck
     - goconst
-    - golint
+    - revive
```

Comment:
there are two `deprecated` warnings for plugin `golint` and `maligned`, and it suggested to use `revive` instead of `golint` and use `maligned` instead of `maligned`.